src/components/container/DailyDataGoal/DailyDataGoal.stories.tsx

```diff
@@ -0,0 +1,147 @@
+import React from "react"
+import Card from "../../presentational/Card"
+import Layout from "../../presentational/Layout"
+import { DailyDataType } from "../../../helpers";
+import DailyDataGoal, { DailyDataGoalProps } from "./DailyDataGoal";
+import { add, startOfDay } from "date-fns";
+
+export default {
+    title: "Container/DailyDataGoal",
+    component: DailyDataGoal,
+    parameters: {
+        layout: 'fullscreen',
+    }
+};
+
+const render = (args: DailyDataGoalProps) => <Layout colorScheme='auto'>
+    <DailyDataGoal {...args} />
+</Layout>;
+
+export const SleepGoal = {
```

Comment:
Are these stories meant to demonstrate a difference in presentation based on dailyDataType or how close someone was/not to their goal. I don't see a difference in the test data generated.
These look more like Goal Achieved, Goal Progress, and Goal No Progress.

Reply:
Basically just exercising a real-world example.  Sleep in this case someone gets credit for even if they just had 1 minute of sleep recorded; whereas wear time only counts if you hit 10 hours etc.